boltdesignsystem/bolt
# DS-298 Tech doc layout design

## Jira

https://pegadigitalit.atlassian.net/browse/DS-298

## Summary

Adds Side Nav component, Content Pagination component, Holy Grail layout, and Article element, and updates TOC component with a transparent variant.

## Details

1. New components: Side Nav (docs commented out) and Content Pagination
2. New layout: Holy Grail (Layout section in PL is now open, but docs commented out)
3. New element: Article
4. Update [Jest config](https://github.com/boltdesignsystem/bolt/pull/2147/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap&file-filters%5B%5D=.twig&hide-deleted-files=true#diff-1e058ca1442e46581b13571fb8d261f9e1f5657e26c96634d4c1072f0f0347f1) to fix broken tests

## How to test

Run the branch locally and check the docs for TOC component, ~~Side Nav component~~ (commented out, WIP), Content Pagination component, ~~Holy Grail layout~~ (commented out, WIP), and Article element. Make sure all props work as stated in the docs.

### Visual changes

TOC component's active indicator's (the teal bar) width is reduced to better reflect information architecture. Side Nav carries more importance than TOC, so TOC's style need to be reduced a little bit.
